### PR TITLE
Create tendrl_definitions_add_object_listing_key.adoc

### PR DESCRIPTION
**Pull request by r0h4n**
tendrl_definitions_add_object_listing_key

### CODE REVIEW

**Comment by r0h4n** [comment]
@Tendrl/tendrl-core , @shtripat @anivargi @nnDarshan @anmolbabu @GowthamShanmugam please review

**Comment by mkudlej** [comment]
Is this specification connected with future specification for tendrl-api for listing objects in etcd?

File: specs/tendrl_definitions_add_object_listing_key.adoc [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+=== Alternatives
+
+None
+
+=== Data model impact:
+
+* Adds "list" key to all Objects in Tendrl namespaces
+
+* The "list" key will point to a relative path in the central store which will contain list of all instances of that object
+eg: 'clusters/xyz/objects/Volumes/' will contain list of all Volumes in that cluster
```
Comment (shtripat) [inline specs/tendrl_definitions_add_object_listing_key.adoc:30]: Shouldnt it be like /clusters/<id>/Volumes rather?

Comment (anivargi) [reply specs/tendrl_definitions_add_object_listing_key.adoc:30]: Ideally it should be /clusters/:cluster_id/volumes, but whatever you put here will be used for listing from the API

Comment (r0h4n) [reply specs/tendrl_definitions_add_object_listing_key.adoc:30]: "objects" is required for https://github.com/Tendrl/specifications/issues/37

**Comment by r0h4n** [comment]
@anivargi any queries on this? I am going to merge this today

**Comment by anivargi** [comment]
@r0h4n No.

**Pull request merged**